NewHorizons/Handlers/InterferenceHandler.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NewHorizons.Handlers
+{
+    public static class InterferenceHandler
+    {
+        private static List<string> _playerInterference;
```

Comment:
you could just store a list of volumes directly instead of strings. comparison would still work since each volume would be a unique instance

Reply:
me when i thought the guid was clever, but yeah that makes sense lol